app/src/main/java/com/cooper/wheellog/WheelData.java

```diff
@@ -1206,7 +1213,10 @@ void decodeResponse(byte[] data, Context mContext) {
         }
         setMaxPwm(mCalculatedPwm);
         if (mWheelType == WHEEL_TYPE.GOTWAY || mWheelType == WHEEL_TYPE.VETERAN) {
-            mCurrent = (int) Math.round(mCalculatedPwm * mPhaseCurrent);
+            setCurrent((int) Math.round(mCalculatedPwm * mPhaseCurrent));
+        }
+        if (mWheelType != WHEEL_TYPE.INMOTION_V2) {
+            setPower(mCurrent * mVoltage);
```

Comment:
тут наверно надо еще разделить на 100 или 
```suggestion
            setPower(getCurrentDouble() * mVoltage);
```

Reply:
Да там чет вчера у меня голова вспухла с этими делениями, хотел сначала тестами пройтись, но оно не работает так. Потом решил тупо протестить что получилось на колесе, но уже поздно было пиликать не хотелось колесами.
Ты там еще добавил теперь вообще ничего не понятно :D протестирую на колесе.

Reply:
Сейчас эти деления сбивают с толку. А метод в decodeResponce не тестируется. Моя логика была, что если суффикс `что-тоDouble`, то оно разделено на 100, как в остальных параметрах.